views/index.sgr

```diff
@@ -1,16 +1,16 @@
 extends(src='layout.sgr')
   block(name='content')
-    div(class='columns')
+    div.columns
       each(loop='product of products')
-        div(class="column")
+        div.column
```

Comment:
One nice sugarml shortcut is that you can use css syntax for setting classes and ids on your elements